lib/backbone.marionette.esm.js

```diff
@@ -846,18 +1235,204 @@ var UIMixin = {
   }
 };
 
-// DomApi
+// Add Feature flags here
+// e.g. 'class' => false
+var FEATURES = {
+  childViewEventPrefix: false,
+  triggersStopPropagation: true,
+  triggersPreventDefault: true,
+  DEV_MODE: false
+};
 
-function _getEl(el) {
-  return el instanceof Backbone.$ ? el : Backbone.$(el);
-} // Static setter
+function isEnabled(name) {
+  return !!FEATURES[name];
+}
 
+function setEnabled(name, state) {
+  return FEATURES[name] = state;
+}
+
+// Event Delegator
+
+function setEventDelegator(mixin) {
```

Comment:
The event delegator handles DOM event delegation and was copied from Backbone.View